Source/WebCore/platform/audio/AudioSession.cpp

```diff
@@ -124,6 +124,8 @@ void AudioSession::removeInterruptionObserver(InterruptionObserver& observer)
 
 void AudioSession::beginInterruption()
 {
+    RELEASE_LOG_ERROR(WebRTC, "AudioSession::beginInterruption %u", m_interruptionObservers.computeSize());
```

Comment:
Should this really be logged as an error?

Reply:
This should be removed, I did it in the new patch